modules/Bio/EnsEMBL/Analysis/Hive/Config/SanityChecksStatic.pm

```diff
@@ -625,19 +604,17 @@ sub _master_config {
       }, # rnaseq_blast
       'layer' => {
         'logic_names' =>    {
-          'genblast'                => 3000,
-          'best_targetted'          => 300,
+          'genblast'                => 100,
+          'best_targetted'          => 100,
```

Comment:
It might be worth having the project_transcripts logic_name here, it's fine that not much from genblast is making it to the layer db, because the rnaseq and projection are "more important"

Reply:
Sure, I will add it, thanks
I noticed that for mammals and primates the layer is quite poor, I am happy to push the changes here too if it is fine for you. Otherwise I can open a new Pr

Reply:
I adjusted the layer in all the clades, I will test these numbers in the next annotations